README.md

```diff
@@ -1,2 +1,110 @@
+
+### Sensors 
+1. The table top is snared with a LDR grid consisting of 4 rows, each row consisting of different no. of LDR. The sensors are placed in determined order.
+2. Rotary encoders are synchronized with each motor. It is used to get the coordinate feedback of defender at every instant.
+
+### Actuators
+Johnson DC motors are used for traversal of the defender over the rods.
+
+### Controllers
+1. Arduino MEGA is used as a microcontroller board.
+2. Cytron motor driver is used to control DC motors.
+
+### Functionality
+1. As soon as a LDR sensor is devoid of room light which means the puck is directly above it, it sends a signal to the microcontroller.
+2. Arduino MEGA is so programmed to identify that particular sensor and instruct the motor driver to start the DC motors to bring the defender in longitudinal alignment with the puck while staying back and hit it with a force as soon as it crosses the first row of sensors.
+
+### Arduino code
+1. We have recognized each sensor as a memory place in a 2D array. We have numbered each LDR and measured its distance from the origin (which is a longitudinal centreline) like a 1D numberline as shown below. Refer to src file.
+![LDR setup figure](https://user-images.githubusercontent.com/68807083/88666808-318fce00-d0fe-11ea-8741-c41be6cc9d71.jpg)
+2. Function 'movemotor' comprises a switch statement and has three arguments. a) a: takes switch case, b) across: takes the transversal displacement, c) forward: takes the longitudinal displacement. This function is called whenever any LDR sends signal of pucks presence and the parameters are passed accordingly to align the defender longitudinaly with the puck. The forward parameter is zero untill the puck crosses row 3. After row 3 is crossed forward parameter has a value and the motor move the defender forward by that value to hit the puck.
+3. Function 'encoder1' and 'encoder2' keeps track of the rotation of motor 1 and motor 2 respectively and feedbacks the coordinates of the defender.      
+
+## Cost structure
+![cost](https://user-images.githubusercontent.com/68807083/88667966-a44d7900-d0ff-11ea-9c41-c59edc657d8f.jpg)
+## Application 
+1.  Gaming parlours
+2.  Shopping malls: can be installed as quick game challenges 
+3.  Hostels: enjoyment for students 
+4.  Houses: as a personal gaming machine
+
+## Limitation 
+1.  The bot cannot exactly predict the trajectory of the puck. And therefore it is unable to hit the puck in a particular direction.
+2.  It is not competent enough to detemine the velocity of the puck so as to hit it with a specific force.
+3.  No plans regarding the introduction of the difficulty level option is executed till now.
+ 
+## Future improvements 
+1.  LCD display module can be installed to interact with the bot like setting up difficulty level, counting scores etc.
+2.  Image processing can be incorporated to contemplate trajectories.
+ 
+## Team members
+1. Mohit Singh (https://github.com/mohitschoudhary)
```

Comment:
@naman99-agar 
Syntax is wrong it should be `` [Mohit Singh](https://github.com/mohitschoudhary)``  [Mohit Singh] which gives (https://github.com/mohitschoudhary)
Do this for all the names